Unix/build.mak

```diff
@@ -237,7 +237,7 @@ distclean:
 
 ifdef BUILD_TESTS
 tests:
-	( LD_LIBRARY_PATH=$(LIBDIR); export LD_LIBRARY_PATH; DYLD_LIBRARY_PATH=$(LIBDIR); export DYLD_LIBRARY_PATH; $(BINDIR)/nits -file:$(TMPDIR)/nitsargs.txt )
+	sudo sh -c "( LD_LIBRARY_PATH=$(LIBDIR); export LD_LIBRARY_PATH; DYLD_LIBRARY_PATH=$(LIBDIR); export DYLD_LIBRARY_PATH; $(BINDIR)/nits -file:$(TMPDIR)/nitsargs.txt )"
```

Comment:
Currently in order to run tests, or even build nits, you have to sudo make ; sudo make tests.
That means that pbuild can't run the regression tests as part of the build.  Moving the sudo to the specific components in the makefiles means they can be built and run without elevation of the make.